modules/ROOT/pages/7.4-release-notes.adoc

```diff
@@ -160,6 +160,13 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== Placing the cursor after a table with a BR element would misplace added newlines before the table instead of after it. 
```

Comment:
It may be good to clarify that it is after the table and before the BR element, not a table with a BR element somewhere inside of it.

Reply:
```suggestion
=== Placing the cursor after a table, but before a BR element would misplace added newlines before the table instead of after it. 
```